gulpfile.js

```diff
@@ -29,19 +29,15 @@ gulp.task('sass', function() {
       .pipe(sassGlob())
       .pipe(sass({
         includePaths: [
-          require('node-normalize-scss').includePaths,
-          'bower_components/sassline/assets/sass',
-          'bower_components/offcanvasmenueffects',
-          'node_modules/susy/sass',
-          'node_modules/breakpoint-sass/stylesheets'
+          // @TODO. Can't get the includes paths to work'
```

Comment:
Can't get includePaths to work, any idea Micha? I'll look into it some more at a later time.